### PR TITLE
Run codespell on code

### PR DESCRIPTION
**Pull request by rhatdan**
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: test/e2e/common_test.go [hunk 1]
```diff
@@ -968,7 +968,7 @@ func populateCache(podman *PodmanTestIntegration) {
 	GinkgoWriter.Printf("-----------------------------\n")
 }
 
-// rmAll removes the direcory and its content,, when running rootless we use
+// rmAll removes the directory and its content,, when running rootless we use
```
Comment (edsantiago) [inline test/e2e/common_test.go:971]: fix double comma too

File: test/apiv2/test-apiv2 [hunk 2]
```diff
@@ -256,7 +256,7 @@ function t() {
         for arg; do
             case "$arg" in
                 # This is just some hack to avoid adding `-d {}` to curl for endpoints where we really need an empty body.
-                # --disable makes curl not lookup the curlrc file, it't should't effect the tests in any way.
+                # --disable makes curl not lookup the curlrc file, it shouldn't effect the tests in any way.
```
Comment (edsantiago) [inline test/apiv2/test-apiv2:259]: Close! You fixed one of my nits, but not the other (should be "affects" with an "a"). I won't block over it; I'm sure the grammar checker will catch it on a future sweep.

**Comment by TomSweeneyRedHat** [comment]
Once @edsantiago 's nits are addressed, LGTM

File: test/system/505-networking-pasta.bats [hunk 3]
```diff
@@ -51,7 +51,7 @@ function pasta_test_do() {
     local bytes="${7}"
 
     # DO NOT ADD ANY CODE ABOVE THIS LINE! Especially skips: that could
-    # lead to the crossreference check not running in CI.
+    # lead to the cross-reference check not running in CI.
```
Comment (edsantiago) [inline test/system/505-networking-pasta.bats:54]: Should you choose to re-push (to fix the "effects" above), could you remove this correction? This code is removed in #19100 so we have a conflict.

File: test/apiv2/test-apiv2 [hunk 4]
```diff
@@ -256,7 +256,7 @@ function t() {
         for arg; do
             case "$arg" in
                 # This is just some hack to avoid adding `-d {}` to curl for endpoints where we really need an empty body.
-                # --disable makes curl not lookup the curlrc file, it't should't effect the tests in any way.
+                # --disable makes curl not lookup the curlrc file, it't shouldn't effect the tests in any way.
```
[the remark this answers is not in the document]
Comment (rhatdan) [reply test/apiv2/test-apiv2]: I love the English language.

**Pull request merged**